index.bs

```diff
@@ -79,30 +72,13 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Proximity Sensor's associated <a>Sensor subclass</a>
-is the {{ProximitySensor}} class.
-
-The Proximity Sensor's associated <a>SensorReading subclass</a>
-is the {{ProximitySensorReading}} class.
+The Proximity Sensor's associated {{Sensor}} subclass is the {{ProximitySensor}} class.
 
 The Proximity Sensor has a <a>default sensor</a>,
 which is the device's main proximity detector.
 
-The Proximity Sensor has a single <a>supported reporting mode</a>
-which is "<a>auto</a>".
-
-The Proximity Sensor's <a>permission</a> name is `"proximity"`.
-It has no <a>associated PermissionDescriptor</a>.
-
-The Proximity Sensor has an associated abstract operation
-to <dfn>retrieve the sensor permission</dfn> which
-must simply return a <a>permission</a> whose name is "proximity".
-
-The Proximity Sensor has an associated abstract operation
-to <dfn lt="Construct SensorReading Object">construct a SensorReading object</dfn>
-which creates a new {{ProximitySensorReading}} object and sets its
-<a attribute for="ProximitySensorReading">distance</a> attribute to
-positive infinity.
+A [=latest reading=] per [=sensor=] of proximity sensors type includes distance,
```

Comment:
A [=latest reading=] per [=sensor=] of proximity sensor type includes three [=map/entries=]
whose [=map/keys=] are "distance", "max", "near" and whose [=map/values=] contain {{ProximitySensor/distance!!attribute}}, {{ProximitySensor/max!!attribute}} and {{ProximitySensor/near!!attribute}} values.